src/Microsoft.Health.Fhir.SqlServer/Features/ChangeFeed/SqlServerFhirResourceChangeDataStore.cs

```diff
@@ -139,6 +139,11 @@ public async Task<IReadOnlyCollection<ResourceChangeData>> GetRecordsAsync(long
                     }
                 }
             }
+            catch (Exception ex) when ((ex is OperationCanceledException || ex is TaskCanceledException) && cancellationToken.IsCancellationRequested)
```

Comment:
Need to update the function exception information on the top at lines 67 and 85.

Reply:
updated the help text with the new exceptions being thrown

Reply:
nit: Need to add the same help text at line 67. There is an overload of the GetRecordsAsync function.

Reply:
updated. thanks